app/views/layouts/embed.html.erb

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Duke Digital Repository Embed</title>
+  <%= stylesheet_link_tag    'application', media: 'all' %>
+  <%= javascript_include_tag 'application' %>
+  <%= csrf_meta_tags %>
+  <%= content_for(:head) %>
+
+  <%# Separate GA account for embeds so as to not bloat visit stats every 
+  time iframe content is served up %>
+  <% if ENV['TRACK_GOOGLE_ANALYTICS'] == 'true' %>
+    <%= render 'google_analytics' %>
```

Comment:
I wonder if it would simplify things to pass in the GA tracking id to the google_analytics partial rather than having separate logic for doing this in a helper?

Is the TRACK_GOOGLE_ANALYTICS env var redundant? Is the presence of the tracking id in the config enough to mean tracking should be on?